docs/time/index.rst

```diff
@@ -415,33 +446,45 @@ that work on `~numpy.ndarray` instances can be used.  E.g., one can reshape
 
 Note that similarly to the `~numpy.ndarray` methods, all but
 :meth:`~astropy.time.Time.flatten` try to use new views of the data,
-with the data copied only if that it is impossible (as discussed, e.g., in
-the documentation for numpy :func:`~numpy.reshape`).
+with the data copied only if that is impossible (as discussed, for example, in
+the documentation for ``numpy`` :func:`~numpy.reshape`).
+
+.. EXAMPLE END
 
 Some arithmetic methods are supported as well: :meth:`~astropy.time.Time.min`,
 :meth:`~astropy.time.Time.max`, :meth:`~astropy.time.Time.ptp`,
 :meth:`~astropy.time.Time.sort`, :meth:`~astropy.time.Time.argmin`,
 :meth:`~astropy.time.Time.argmax`, and :meth:`~astropy.time.Time.argsort`.
-E.g.::
+
+.. EXAMPLE START: Reshaping Time Instances Using Arithmetic Methods
+
+To reshape |Time| instances using arithmetic methods::
 
   >> t.max()
   <Time object: scale='utc' format='mjd' value=50002.5>
   >> t.ptp(axis=0)  # doctest: +FLOAT_CMP
   <TimeDelta object: scale='tai' format='jd' value=[2. 2.]>
 
+.. EXAMPLE END
+
 .. _astropy-time-inferring-input:
 
-Inferring input format
+Inferring Input Format
 ^^^^^^^^^^^^^^^^^^^^^^
 
-The |Time| class initializer will not accept ambiguous inputs,
-but it will make automatic inferences in cases where the inputs are
-unambiguous.  This can apply when the times are supplied as `~datetime.datetime`
-objects or strings.  In the latter case
-it is not required to specify the format because the available
-string formats have no overlap.  However, if the format is known in advance
-the string parsing will be faster if the format is provided.
-::
+The |Time| class initializer will not accept ambiguous inputs, but it will make
+automatic inferences in cases where the inputs are unambiguous. This can apply
+when the times are supplied as `~datetime.datetime` objects or strings. In the
```

Comment:
Here, again, I fear the documentation got a bit out of date: it should be "... objects, inputs for ``ymdhms``, or strings."